__tests__/server.test.js

```diff
@@ -53,6 +53,7 @@ describe("/api/article/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "11",
```

Comment:
if we favour `objectContaining` matcher over `toEqual`, then there's no need to edit old tests;
Think back to the open-closed principle: open to extension, closed to modification. 

This more something to keep in mind, rather than a refactor suggestion, I suppose. 🧠

Reply:
sure thing thanks